docs/docs/changelog/next.md

```diff
@@ -6,7 +6,7 @@ tags: [release, eCalc]
 sidebar_position: 1
 ---
 
-# eCalc
+# eCalc v8.7

```

Comment:
this should be in the `v8-7.md` file, as we have created that release..or should already have been in the 8.6 version, but we forgot to update the changelog. v8-7.md should exists, nevertheless...

Reply:
Next now, is to the potential v8.8 release

Reply:
Things are updated now, with changes in the v8-7.md file.